lib/ansible/modules/cloud/packet/packet_device.py

```diff
@@ -22,17 +22,17 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: packet_device
 
-short_description: create, destroy, start, stop, and reboot a Packet Host machine.
+short_description: create, destroy, start, stop, and reboot a bare metal server in the Packet Host.
 
 description:
-    - create, destroy, update, start, stop, and reboot a Packet Host machine. When the machine is created it can optionally wait for it to have an
-      IP address before returning. This module has a dependency on packet >= 1.0.
-    - API is documented at U(https://www.packet.net/help/api/#page:devices,header:devices-devices-post).
+    - create, destroy, update, start, stop, and reboot a bare metal server in the Packet Host (a "device" in the API terms).
```

Comment:
'manage' should cover all actions, no need to be this explicit in the description